Sources/VLCEmptyLibraryView.m

```diff
@@ -0,0 +1,41 @@
+/*****************************************************************************
+ * VLCEmptyLibraryView.m
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Mike JS. Choi <mkchoi212 # icloud.com>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+#import <Foundation/Foundation.h>
+#import "VLCEmptyLibraryView.h"
+#import "VLCFirstStepsViewController.h"
+
+@implementation VLCEmptyLibraryView
+
++ (instancetype)init
```

Comment:
This should probably be initWithCoder and you shouldn't have an instance of VLCEmptyView in it's own init, also no asserts but give the view in the xib an identifier and you load the nib with identifier

Reply:
that should remove the whole looking through subviews and you just got one instance but this code belongs in the mediaviewcontroller. Also New Classes should be in Swift 😀